File: FrontEnd/src/app/features/reservations/shipCrews/classes/view-models/shipCrew-list-vm.ts
import { SimpleEntity } from './../../../../../shared/classes/simple-entity'
export interface ShipCrewListVM {

    id: number
    ship: SimpleEntity
    specialty: SimpleEntity
    lastname: string
    firstname: string
    birthdate: string
    formattedBirthdate: string
    isActive: boolean

}

File: API/Features/Reservations/ShipCrews/ViewModels/ShipCrewListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Reservations.ShipCrews {

    public class ShipCrewListVM {

        public int Id { get; set; }
        public SimpleEntity Ship { get; set; }
        public SimpleEntity Specialty { get; set; }
        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public string Birthdate { get; set; }
        public bool IsActive { get; set; }

    }

}